scripts/remove_invalid_social_entries.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
This should be a proper Django migration. See examples of custom migrations in the `osf/migrations/` directory